.gitlab/e2e/e2e.yml

```diff
@@ -124,21 +127,26 @@ k8s-e2e-otlp-main:
       annotations:
         - $EXTERNAL_LINKS_PATH
 
+.new_e2e_go_deps:
+  - go_e2e_deps
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I would keep the original name for one single dependency

Reply:
sounds good to me! :onit:

Reply:
Done, it's more generic and we can add more things to it later if needed